<?xml version="1.0"?>
<launch>
   <node pkg="lively_ik" type="generate_info_file.py" name="generate_info_file" output="screen"/>
</launch>
